pkg/parse.go

```diff
@@ -18,7 +18,7 @@ import (
 	"golang.org/x/tools/go/packages"
 )
 
-var autoGeneratedRegex = regexp.MustCompile(`^\s*Code generated .* DO NOT EDIT\.(\n)?$`)
+var autoGeneratedRegex = regexp.MustCompile(`Code generated .* DO NOT EDIT\.`)
```

Comment:
The way I would prefer to do this is instead loop over every line in the comment group.

https://github.com/vektra/mockery/blob/103f702b7d55c05bd13c9ee4dd25a60a9d31cac3/pkg/parse.go#L442

The [`ast.CommentGroup`](https://pkg.go.dev/go/ast#CommentGroup) can be iterated over, so it's probably better to leave this regex as-is and just run it against each specific `ast.Comment`.

Reply:
Ok, I changed it. I now had to include the `//` in the regex, as the `Text()` function to strip it is only available on the `ast.CommentGroup`, not on individual `Comment` and I didn't want to re-implement this logic. I believe it should be fine as the generated comments always follow that format, e.g. are never using `/* ... */`